example/README.md

```diff
@@ -8,23 +8,23 @@ yarn dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+Flag in use is `nextjs-example`. https://app.unleash-hosted.com/demo/projects/default/features/nextjs-example
```

Comment:
What does this mean? What's the link there for? Where is the flag in use?

Reply:
If you run the example without any modifications, it's connected to our demo.

Reply:
got it! I know you merged it already, but I would suggest writing that out explicitly in the readme. Something like this?

```suggestion
The demo relies on a flag called `nextjs-example`. By default, it will connect to the [Unleash demo instance](https://app.unleash-hosted.com/demo/projects/default/features/nextjs-example).
```